core_java/twitter/src/test/java/ca/jrvs/apps/twitter/service/TwitterServiceIntTest.java

```diff
@@ -0,0 +1,104 @@
+package ca.jrvs.apps.twitter.service;
+
+import ca.jrvs.apps.twitter.dao.TwitterDao;
+import ca.jrvs.apps.twitter.dao.TwitterHttpHelper;
+import ca.jrvs.apps.twitter.dao.helper.HttpHelper;
+import ca.jrvs.apps.twitter.model.Coordinates;
+import ca.jrvs.apps.twitter.model.Tweet;
+import ca.jrvs.apps.twitter.util.JsonUtil;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+public class TwitterServiceIntTest {
+
+    //Create a private twitterService to test
+    private static TwitterService twitterService;
+    //Create a list of Ids
+    private List<String> IdList = new ArrayList<String>();
+
+    //Setup and also post tweet so that we can get it during our test later (and then also delete at the end)
+    @Before
+    public void setUp() throws JsonProcessingException {
+
+        String CONSUMER_KEY = System.getenv("consumerKey");
+        String CONSUMER_SECRET = System.getenv("consumerSecret");
+        String ACCESS_TOKEN = System.getenv("accessToken");
+        String TOKEN_SECRET = System.getenv("tokenSecret");
+
+        System.out.println(CONSUMER_KEY + " | " + CONSUMER_SECRET + " | " + ACCESS_TOKEN
+                + " | " + TOKEN_SECRET);
+
+        HttpHelper httpHelper = new TwitterHttpHelper(CONSUMER_KEY, CONSUMER_SECRET, ACCESS_TOKEN, TOKEN_SECRET);
+
```

Comment:
comments are not necessary. please go through all your entire code base and remove unnecessary comments and empty lines